sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -533,6 +533,17 @@ export class EventHubSender extends LinkEntity {
             return reject(translate(e));
           };
 
+          if (options && options.cancellationToken) {
+            options.cancellationToken.onabort = () => {
```

Comment:
When the send operation is successful, we should remove the listener that we added here. Can you check how the Aborter class allows us to do that?